extractor/src/main/java/org/schabi/newpipe/extractor/services/rumble/RumbleParsingHelper.java

```diff
@@ -218,4 +218,18 @@ public static synchronized Map<String, List<String>> getMinimalHeaders() {
         }
         return HEADERS;
     }
+
+    public static String getEmbedVideoId(final String rb) {
+        final String VALID_URL = "https?://(?:www\\.)?rumble\\.com/embed/(?:[0-9a-z]+\\.)?(?<id>[0-9a-z]+)";
+        final String EMBED_REGEX = "(?:<(?:script|iframe)[^>]+\\bsrc=|[\"']embedUrl[\"']\\s*:\\s*)[\"'](?<url>" + VALID_URL + ")";
+        Pattern pattern = Pattern.compile(EMBED_REGEX);
+        Matcher matcher = pattern.matcher(rb);
+        if (matcher.find()) {
+            String url = matcher.group("url");
```

Comment:
I like the group naming feature but as this feature is not available on older android versions (and desugaring does not support it) could you please use the matching group number instead?